alphagov/stagecraft
# Add an admin action to clone selected collectors

## What

* Added an admin action "Clone collector" to clone selected collectors, to make it easy to create a new collector based on an existing collector's attributes

## How to review

1. Browse to Collectors admin page (/admin/collectors/collector/)
2. Select one or more collectors to clone, by checking their checkboxes
3. Select "Clone collector" from the "Action" dropdown and click on "Go"
4. a. If one collector was selected, be taken to the edit page for the new clone
   b. If multiple collectors were selected, see that new collectors are now listed with "-clone" in the slug
